README.md

```diff
@@ -126,41 +123,108 @@ helm delete my-galaxy
+| `service.nodePort`                        | If `service.type` is set to `NodePort`, then this can be used to set the port at which Galaxy will be available on all nodes' IP addresses |
+| `workflowHandlers.*`                      | Configuration for the workflowHandlers (See below for all options) |
+| `webHandlers.*`                           | Configuration for the webHandlers                            |
+| `jobHandlers.*`                           | Configuration fo the jobHandlers                             |
+| `jobHandlers.priorityClass.enabled`       | Assign a [priorityClass](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass) to this handler. |
+| `jobHandlers.priorityClass.existingClass` | The [priorityClass](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass) to assign. |
+| `metrics.enabled`                         | Enable metrics gathering.                                    |
+| `metrics.image.repository`                | The location of the [galay-metrics-scraping](https://github.com/CloudVE/galaxy-docker-k8s-metrics) image to use. |
+| `metrics.image.tag`                       | The image version to use.                                    |
+| `metrics.image.pullPolicy`                | Define the [pull policy](https://kubernetes.io/docs/concepts/containers/images/#updating-images), that is, when Kubernetes will pull the image. |
+| `serviceAccount.create`                   | The serviceAccount will be created if it does not exist.     |
+| `serviceAccount.name`                     | The serviceAccount account to use.                           |
+| `rbac.enabled`                            | Enable Galaxy job RBAC                                       |
+| `securityContext.fsGroup`                 | The [group](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) for any files created. |
+| `persistence.enabled`                     | Enable persistence using PVC                                 |
+| `persistence.name`                        | Name of the PVC                                              |
+| `persistence.accessMode`                  | PVC access mode for the Galaxy volume                        |
+| `persistence.size`                        | PVC storage request for the Galaxy volume, in GB             |
+| `persistence.mountPath`                   | Path where to mount the Galaxy volume                        |
+| `extraInitCommands`                       | Extra commands that will be run during initialization.       |
+| `extraEnv`                                | Environment variables that will be defined                   |
+| `ingress.enabled`                         | Enable Kubernetes ingress                                    |
+| `ingress.canary.enabled`                  | Enable [canary](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#canary) ingress. |
```

Comment:
Although we use similar terminology and the purpose is related, it's not using the built-in nginx features, so I've rephrased as follows.
```suggestion
| `ingress.canary.enabled`                  | This will create an additional ingress for detecting activity on Galaxy. Useful for autoscaling on activity. |
```